### DISCUSSION TITLE
osmand 3.9.5 crashes on iphone 8 ios 14.4.1

### DISCUSSION THREAD

**Discussion by mayday-m** [osmandapp/OsmAnd#11121] [Q&A] [answered]
Since the latest update, Osmand crashes on startup without a notification. What can i do about this?

**Comment by wordplay77** [comment, marked as answer]
Looks like it’s fixed for me in 3.9.7 just updated

**Reply by mayday-m** [reply]
yes, it's fixed in 3.9.7! Thanks!
